jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwKeyInput.java

```diff
@@ -87,6 +87,16 @@ public void invoke(long window, int key, int scancode, int action, int mods) {
 
                 keyInputEvents.add(event);
             }
+
+            @Override
+            public void close() {
```

Comment:
A build error is produced otherwise, because another interface declares these methods as `public void close() throws IOException` which overwrites the declaration of the super interface where it delegates to. So I have to overwrite these methods, because the no-throw version is needed.

P.S. / offtopic: I'm always confused: override or overwrite? :)